cdap-docs/_common/common-build.sh

```diff
@@ -304,6 +304,7 @@ function download_file() {
 
   echo "Downloading using curl ${file_name}"
   echo "from ${source_dir}"
+  echo "curl --silent ${source_dir}/${file_name} --output ${target}"
```

Comment:
Did you intend to keep this or it just for debugging?

Reply:
This would be helpful. Until I added this check the I couldn't figure out why the call to fetch README.rst was failing and generating an incorrect hash. 
The original logic was adding extra `/` which made github return 301 (Permanently moved) which was causing the build to fail for some reason.

Reply:
I see, thanks for explaining. I guess we need to pass -L arg to curl to follow redirects